pydicom/tests/test_dataelem.py

```diff
@@ -578,3 +577,34 @@ def test_unknown_vr(self):
                              0, False, True)
         with pytest.raises(NotImplementedError):
             DataElement_from_raw(raw, default_encoding)
+
+    @pytest.fixture
+    def enable_convert_wrong_length_to_UN(self):
+        old_value = config.convert_wrong_length_to_UN
+        config.convert_wrong_length_to_UN = True
+        yield
+        config.convert_wrong_length_to_UN = old_value
+
+    def test_wrong_bytes_length_exception(self):
+        """Check exception when number of raw bytes is not correct."""
+        raw = RawDataElement(Tag(0x00190000), 'FD', 1, b'1', 0, False, True)
```

Comment:
Technically, this should ensure that the config flag is False, but I don't see us ever making that the default, so we can leave it as is.

Reply:
I went ahead and made this change since I already needed to cleanup <code>test_dataelem.py</code>.